### DISCUSSION TITLE
Estimating lake residence times (Task 2.4)

### DISCUSSION THREAD

**Discussion by JamesSample** [NIVANorge/teotil3#2] [Ideas]
We first need to identify the lakes along the main river stem in each regine catchment. It looks as though this can be achieved by intersecting the lakes dataset with the "hovedelv" polylines, but this needs more investigation.

NVE has at least some depth data for 686 lakes (see `Innsjo_Dybdedata.shp`). Of these, residence times in years are provided explicitly for 386 lakes (see the `oppholdTid` field in the attribute table).

Mean water depth and lake area are available for 423 lakes, which could be used to estimate mean depth from area for all lakes along the main stem (as done in the original model). Additional covariates could perhaps be included to improve this relationship.

NVE's regine dataset includes a column named `totTilsig`, which seems to be long-term average (1961-90) annual discharge for each regine. This could be used to represent the flow along the main stem in each catchment, which can then be combined with lake volume estimates to calculate residence times. Results should be compared to values for the 386 lakes with explicit data (see above).

**Comment by JamesSample** [comment]
In addition to the NVE data described above, depth measurements were taken during the 2019 "1000 Lakes" survey (see the attachment below and e-mail from Øyvind K received 03.06.2022 at 10.53)

[1000_lakes_2019_depth_data.xlsx](https://github.com/NIVANorge/teotil3/files/8831310/1000_lakes_2019_depth_data.xlsx)

I assume these are the depths measured at each sampling point, rather than the maximum depth for each lake. Is that right, @okaste?

**Comment by okaste** [comment]
I think they tried to put the sampling point above what they assumed to be the deepest part of the lake. But anyway, the numbers should only be considered as rough estimates for max depth.
